assets/scss/base.scss

```diff
@@ -184,3 +184,157 @@ svg.svg-inline--fa{
 .tinymce-textarea {
   background-color: $backgroundWhite;
 }
+
+
+body {
+  font-size: .875rem;
+}
+
+.feather {
+  width: 16px;
+  height: 16px;
+  vertical-align: text-bottom;
+}
+
+/*
+ * Sidebar
+ */
+
+.sidebar {
+  position: fixed;
+  top: 0;
+
+  /* rtl:raw:
+  right: 0;
+  */
+  bottom: 0;
+
+  /* rtl:remove */
+  left: 0;
+  z-index: 100;
+
+  /* Behind the navbar */
+  padding: 48px 0 0;
+
+  /* Height of navbar */
+  box-shadow: inset -1px 0 0 rgba(0, 0, 0, 0.1);
+}
+
+@media (max-width: 767.98px) {
```

Comment:
Please use variables to clarify why it's this specific number. I guess it makes sense, but I also can't tell for sure.